OWML.ModHelper/ModManifest.cs

```diff
@@ -41,5 +41,11 @@ public class ModManifest : IModManifest
 
         [JsonIgnore]
         public string AssemblyPath => ModFolderPath + Filename;
+
+        [JsonProperty("minimalGameVersion")]
+        public string MinimalGameVersion { get; private set; } = "";
+
+        [JsonProperty("maximalGameVersion")]
+        public string MaximalGameVersion { get; set; } = "";
```

Comment:
I prefer MinGameVersion and MaxGameVersion, but if you wanna write it out, Minimum and Maximum is better.

Reply:
I too feel that `-imal` is a bit excessive